### PR TITLE
Update go/mio README.md

### PR DESCRIPTION
**Pull request by mengwanguc**
I'm using go 1.15.5 on CenOS 7.8.2003.
My `go env GOBIN` doesn't show any output.
My `go env GOPATH` points to `/home/cc/go` and I can find the `mcp` binary installed in `GOPATH/bin`.

Therefore update the README to help users locate their binary.

A similar issue was reported in #682

-----
[View rendered bindings/go/README.md](https://github.com/mengwanguc/cortx-motr/blob/patch-10/bindings/go/README.md)

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Comment by mengwanguc** [comment]
Hello, can someone review this PR?
@andriytk , can you take a look? It's about the documentation of go bindings.

Thank you very much,
Meng

File: bindings/go/README.md [hunk 1]
```diff
@@ -62,5 +62,8 @@ cd motr/bindings/go/mcp && go build && go install
 The binary will be installed to your `GOBIN` directory
 (check with `go env GOBIN`).
```
Comment (andriytk) [inline bindings/go/README.md:63]: Just add here: "or `echo $(go env GOPATH)/bin`" and that will be enough.

**Review by andriytk — CHANGES_REQUESTED** [review]

File: bindings/go/README.md (hunk 1 above)
Comment (mengwanguc) [reply bindings/go/README.md]: Thanks Andriy! I've updated the commit

**Review by andriytk — APPROVED** [review]

**Comment by mengwanguc** [comment]
Hello, can someone else review this PR? Thank you very much!

**Comment by andriytk** [comment]
@madhavemuri, could you review and approve it please?

**Review by mehjoshi — APPROVED** [review]

**Comment by mehjoshi** [comment]
# go env GOPATH
/root/go
# go env GOBIN

# go build

# go install
# ls -al /root/go
total 12
drwxr-xr-x   3 root root 4096 Jul 22 07:38 .
dr-xr-x---. 11 root root 4096 Jul 22 07:38 ..
drwxr-xr-x   2 root root 4096 Jul 22 07:38 bin

# ls -al /root/go/bin
total 2736
drwxr-xr-x 2 root root    4096 Jul 22 07:38 .
drwxr-xr-x 3 root root    4096 Jul 22 07:38 ..
-rwxr-xr-x 1 root root 2789664 Jul 22 07:38 mcp

# echo $(go env GOPATH)/bin
/root/go/bin

# pwd
/workspace/sandbox/cortx-motr/bindings/go/mcp

Verified the change.

**Pull request merged**